bin/nuxt-build

```diff
@@ -51,6 +51,10 @@ if (argv.analyze && typeof options.build.analyze !== 'object') {
   options.build.analyze = true
 }
 
+if (options.build.analyze) {
+  consola.warn('Notice: Please do not deploy bundles built with analyze mode, it\'s only for analyzing purpose')
```

Comment:
This may be shown for `nuxt-generate` command too. Maybe refactor it into Builder or just duplicate.

Reply:
Good idea